tests/mocks/test_id_site.py

```diff
@@ -31,14 +31,15 @@ def setUp(self):
     def test_building_id_site_redirect_uri(self):
```

Comment:
Could you turn this into a separate test? I want to keep the old one there, and assert that usd doesn't exist it there by default.